azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/LocalizationConfiguration.java

```diff
@@ -31,16 +35,26 @@
  * @see CallComposite
  */
 public final class LocalizationConfiguration {
-    private int layoutDirection;
+    private final Integer layoutDirection;
     private final Locale locale;
 
+    /**
+     * Create Localization configuration.
+     *
+     * @param locale The {@link Locale}; eg,. Locale.US
+     */
+    public LocalizationConfiguration(@NonNull final Locale locale) {
+        this.locale = locale;
+        this.layoutDirection = LayoutDirection.LTR;
```

Comment:
As we decided to make layoutDirection nullable, we do not need to set default value. Same for line 69, 91

Reply:
Got it. Also updated the java class links as well.